charts/logging/alloy/values.yaml

```diff
@@ -0,0 +1,513 @@
+            target_label  = "job"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_namespace"]
+            target_label  = "namespace"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_name"]
+            target_label  = "instance"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_container_name"]
+            target_label  = "container_name"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_uid", "__meta_kubernetes_pod_container_name"]
+            separator     = "/"
+            target_label  = "__path__"
+            replacement   = "/var/log/pods/*$1/*.log"
+          }
+        }
+
+        local.file_match "kubernetes_pods_name" {
+          path_targets = discovery.relabel.kubernetes_pods_name.output
+        }
+
+        loki.process "kubernetes_pods_name" {
+          forward_to = [loki.write.default.receiver]
+
+          stage.cri { }
+        }
+
+        loki.source.file "kubernetes_pods_name" {
+          targets               = local.file_match.kubernetes_pods_name.targets
+          forward_to            = [loki.process.kubernetes_pods_name.receiver]
+          legacy_positions_file = "/run/alloy/positions.yaml"
+        }
+
+        discovery.relabel "kubernetes_pods_app" {
+          targets = discovery.kubernetes.kubernetes_pods_name.targets
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_label_name"]
+            regex         = ".+"
+            action        = "drop"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_label_app"]
+            target_label  = "__service__"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_node_name"]
+            target_label  = "__host__"
+          }
+
+          rule {
+            source_labels = ["__service__"]
+            regex         = ""
+            action        = "drop"
+          }
+
+          rule {
+            regex  = "__meta_kubernetes_pod_label_(.+)"
+            action = "labelmap"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_namespace", "__service__"]
+            separator     = "/"
+            target_label  = "job"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_namespace"]
+            target_label  = "namespace"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_name"]
+            target_label  = "instance"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_container_name"]
+            target_label  = "container_name"
+          }
+
+          rule {
+            source_labels = ["__meta_kubernetes_pod_uid", "__meta_kubernetes_pod_container_name"]
+            separator     = "/"
+            target_label  = "__path__"
+            replacement   = "/var/log/pods/*$1/*.log"
```

Comment:
I think we can get rid of the first star here, right? The path is deterministic or?